app/assets/stylesheets/components/_game-card.scss

```diff
@@ -0,0 +1,98 @@
+.game-card{
+  @media (max-width: 768px) {
+    display: none,
+  }
+
+    margin : auto;
+    margin-bottom: 3rem;
+    max-width: 800px;
+    max-height: 220px;
+    display: flex;
+
+
+  .picture{
+    height: 200px;
+    width : 315px;
+  }
+
+  img{
+    object-fit: cover;
+    border-radius: 3px;
+  }
+
+}
+
+.game-content, .game-content-mobile{
+  margin-left: 1rem;
+  font-size: $body-font;
+  font-size: 16px;
+  font-weight: 400;
+
+  h4{
+    font-size: 20px;
+    font-family: $headers-font;
+    margin-bottom: 1,5rem;
+  }
+
+  p{
+    margin: 0;
+  }
+  span{
+      color: gray;
+      font-weight: 300;
+    }
+
+}
+
+.game-published{
+
+  padding: 1rem 0px;
+
+  font-size : 12px ;
+}
+
+.game-card-mobile{
+  background: $lightest-gray;
+
+  @media (min-width: 768px){
+    display: none;
+  }
+
+  }
+  .game-content-mobile{
+
+    margin-top: 20px;
+    margin-bottom: 80px;
+  }
+
+  img{
+    width : 95%;
+    height: 350px;
+    margin: 15px;
+    object-fit: cover;
+    border-radius: 3px;
+  }
+  .picture-mobile{
+    height: 350px;
+  }
+
+  .title-mobile{
+    padding: 10px 0px;
+  }
+
+h4{
+  color: #f2f2f2;
+  text-shadow: 0px 1px 5px #a7a7a7;
```

Comment:
Don't understand this. Why do you want to add text shadow?

Reply:
<img width="671" alt="Capture d’écran 2019-12-30 à 15 08 31" src="https://user-images.githubusercontent.com/49406128/71585351-83e02080-2b16-11ea-8d69-437b79149ce1.png">
<img width="670" alt="Capture d’écran 2019-12-30 à 15 09 21" src="https://user-images.githubusercontent.com/49406128/71585364-93f80000-2b16-11ea-9f69-a9856cce90ad.png">

I thought, in the mobile version, it was more "soft" to have lightening under the white text, I gave you screenshots with or without, tell me :)

Reply:
mmmh OK I see. Not a bad idea, but for the moment I see the text with shadow on desktop version, like on the screenshot above

Reply:
Ok looks way better now, thanks for your work 🙏
Keep it like this for the moment, we'll ask to Max which version he prefer